src/raftstore/store/debug.rs

```diff
@@ -79,6 +78,35 @@ impl Debugger {
             Err(e) => Err(box_err!(e)),
         }
     }
+
+    pub fn size<'a>(&self, region_id: u64, cfs: &[&'a str]) -> Result<Vec<(&'a str, usize)>> {
```

Comment:
Or it takes `cfs: &[String]`, service side[1] would be much simpler. 

[1] https://github.com/pingcap/tikv/pull/2329/files#diff-a0345b86052a238929c03760ee459f0aR126

Reply:
maybe we can use Into<String> ?